extensions/datastores/accumulo/src/main/java/mil/nga/giat/geowave/datastore/accumulo/query/AccumuloConstraintsQuery.java

```diff
@@ -33,12 +43,15 @@
 	protected final List<DistributableQueryFilter> distributableFilters;
 	protected boolean queryFiltersEnabled;
 
+	protected final Pair<DataAdapter<?>, DataStatistics<?>> computeStats;
```

Comment:
maybe, but we do want to be able to support this for all *query* AccumuloConstraintsQueries, not delete though